features/page_objects/frontEnd/confirmActivitiesAndAssessments.page.js

```diff
@@ -0,0 +1,94 @@
+const FrontEndPageObject = require('./base/FrontEndPageObject').FrontEndPageObject
+let i
+let assessmentType
+let assessmentPrice
+
+class ConfirmActivitiesAndAssessmentsPage extends FrontEndPageObject {
+  get title () { return 'Confirm activities and assessments' }
+  get totalText () { return { id: 'total-desc' } }
+  get totalPrice () { return { id: 'total-cost' } }
+
+  findAssessments (applicationNumber) { return { id: 'cost-item-' + applicationNumber + '-desc' } }
```

Comment:
I will approve but the current best practice for javascript is to use string templates using the back-tick. eg:
findAssessments (applicationNumber) { return { id: `cost-item-${applicationNumber}-desc` } }